content/docs/howto/encode-json-yaml-with-cue/en.md

```diff
@@ -3,9 +3,17 @@ title: Encode JSON or YAML with CUE
 weight:
 draft: false
 toc_hide: true
+date: "2023-08-22"
 tags:
     - Use encodings in CUE
+authors:
```

Comment:
Not sure if we should already add this to existing content. Maybe only add it to the blog example and the front-matter example pages? Or we can add it to an existing docs page for testing but we need to check with Paul @myitcv  if we've added the right authors. Also I don't think reading time is relevant here (should be supported here but I don't think it will be just on docs that often.)

Reply:
For now it's to show how it all looks. If needed all can be removed before merging this PR @myitcv 
Of course, we can also add the correct values, but then I need to get them first ;)

Reply:
Thanks for flagging. We can leave for now, and we will tidy up later.